samples/javascript_nodejs/21.corebot-app-insights/dialogs/flightBookingRecognizer.js

```diff
@@ -4,10 +4,17 @@
 const { LuisRecognizer } = require('botbuilder-ai');
 
 class FlightBookingRecognizer {
-    constructor(config) {
+    constructor(config, telemetryClient) {
         const luisIsConfigured = config && config.applicationId && config.endpointKey && config.endpoint;
         if (luisIsConfigured) {
-            this.recognizer = new LuisRecognizer(config, {}, true);
+            // Set the recognizer options depending on which endpoint version you want to use e.g v2 or v3.
+            // More details can be found in https://docs.microsoft.com/en-gb/azure/cognitive-services/luis/luis-migration-api-v3
```

Comment:
Need to remove the local from the URL links.

```suggestion
            // More details can be found in https://docs.microsoft.com/azure/cognitive-services/luis/luis-migration-api-v3
```